### ISSUE TITLE
Blog Stats Widget: Change Label on Hits (12,345) Text Field

### ISSUE THREAD

**Issue by mdrockwell** [Automattic/jetpack#16182]
#### Steps to reproduce the issue

1. Ensure Site Stats module is active
2. Navigate to Customize > Widgets > WIDGET LOCATION
3. Click on "Add a Widget"
4. Select the Blog Stats widget to add

#### What I expected
The Hits text field should have a more descriptive label, such as "Number Value", "Pageviews Value", "Pageview Description", or something similar.

#### What happened instead
The text field is labeled with "12,345", which doesn't describe what the field is.

**Screenshots**
![Screen Shot 2020-06-17 at 13 41 39](https://user-images.githubusercontent.com/42454301/84931447-cdaacc00-b0a0-11ea-9c63-bb2fcb67c56d.png)
